src/backend/cpu/CMakeLists.txt

```diff
@@ -47,11 +47,24 @@ IF(NOT UNIX)
     ADD_DEFINITIONS(-DAFDLL)
 ENDIF()
 
+INCLUDE(ExternalProject)
```

Comment:
The external project build commands have their own files which are in CMakeModules. Although it will be called only from the CPU backend, the cmake file should be in CMakeModules. That's how we are going about with clBLAS/clFFT as well.